### PR TITLE
New Adapter: Aceex

### PR DESCRIPTION
**Pull request by supportAceex**

### CODE REVIEW

File: static/bidder-info/aceex.yaml
Comment (SyntaxNode) [inline static/bidder-info/aceex.yaml:3]: If you are not registered with IAB Europe for GDPR, please remove this line entirely instead of setting it to 0.

Comment (supportAceex) [reply static/bidder-info/aceex.yaml]: @SyntaxNode Thank you for your review! Fixed

File: adapters/aceex/aceex.go
Comment (guscarreon) [inline adapters/aceex/aceex.go:182]: The current logic makes `getMediaTypeForImp` to return `openrtb_ext.BidTypeBanner` even if we didn't find a matching `imp.ID` in line 184. Other adapters throw an error if the `imp.ID` didn't find a match. Does that make sense in the context of your adapter? This is not a requirement, I'm ok either way.

**Review by guscarreon — COMMENTED** [review]
Code looks pretty solid and very well covered by unit tests. Left a couple of questions

File: adapters/aceex/aceex.go
Comment (supportAceex) [reply adapters/aceex/aceex.go]: fixed!

File: adapters/aceex/aceex.go
Comment (supportAceex) [reply adapters/aceex/aceex.go:114]: fixed!

**Review by guscarreon — COMMENTED** [review]
@supportAceex thank you for adressing our feedback. Your code looks pretty good but test case `adapters/aceex/aceextest/supplemental/status-code-no-content.json` began failing in this last commit. Can you please correct before we approve?

**Review by AlexBVolcy — COMMENTED** [review]
I also think the code looks solid overall, but like Gus mentioned, there's a test that needs to be fixed before approval

**Comment by supportAceex** [comment]
Thank you! Fixed!

**Review by guscarreon — APPROVED** [review]
LGTM

**Review by hhhjort — APPROVED** [review]
LGTM

**Pull request merged**
